asllib/doc/Statements.tex

```diff
@@ -3071,28 +3071,79 @@ \subsubsection{SemanticsRule.WriteFolder\label{sec:SemanticsRule.WriteFolder}}
 \section{Print Statements\label{sec:PrintStatements}}
 \subsection{Syntax}
 \begin{flalign*}
-\Nstmt \derives \ & \Tprint \parsesep \Plist{\Nexpr} \parsesep \Tsemicolon &
+\Nstmt \derives \ & \Tprint \parsesep \Plist{\Nexpr} \parsesep \Tsemicolon & \\
+\Nstmt \derives \ & \Tprintln \parsesep \Plist{\Nexpr} \parsesep \Tsemicolon & \\
 \end{flalign*}
 
 \subsection{Abstract Syntax}
 \begin{flalign*}
-\stmt \derives\ & \SPrint(\overtext{\expr^*}{args}, \overtext{\Bool}{debug}) &
+  \stmt \derives\ & \SPrint(\overtext{\expr^*}{args}, \overtext{\Bool}{newline}, \overtext{\Bool}{debug}) &
```

Comment:
Is the `debug` field used (i.e. set to `TRUE`) in the reference document? I'm wondering if it is worth removing if not

Reply:
Let's remove it.